pages/docs/reference/whatsnew14.md

```diff
@@ -612,3 +612,34 @@ Complete the initial configuration, and when you add a new dependency to `cocoap
 The new dependency will be added automatically. No additional steps are required.
 
 Learn [how to add dependencies](native/cocoapods.html).
+
+## Scripting and REPL
+
+### New dependencies resolution API
+
+In 1.4, we present the new API for dependency resolution and its implementations. It is published in new artifacts
```

Comment:
Now it seems to mee that it needs a bit of clarification. What about replacing "dependency resolution" in this line with something like "external dependencies resolution (e.g. Maven artifacts)"?